apps/frontend/src/pages/access-management/access-management.tsx

```diff
@@ -83,6 +100,38 @@ const AccessManagement = (): JSX.Element => {
 		[dispatch, onModalClose, userPage, userPageSize],
 	);
 
+	const handleGroupDelete = useCallback((): void => {
+		if (hasSelectedGroup) {
+			void dispatch(
+				groupActions.deleteById({
+					groupQuery: { page: groupPage, pageSize: groupPageSize },
+					id: selectedGroupId as number,
+					userQuery: { page: userPage, pageSize: userPageSize },
```

Comment:
we can remove groups/users dispatch inside this action
```suggestion
					id: selectedGroupId as number,
```
I want it to be done like this

```tsx
	useEffect(() => {
		if (groupDeleteStatus === DataStatus.FULFILLED) {
			onGroupPageChange(INITIAL_PAGE);
	        onUserPageChange(INITIAL_PAGE);
		}
	}, [projectPatchStatus, onUserPageChange, onGroupPageChange]);
```

Reply:
Sorry, I'm having trouble understanding what `INITIAL_PAGE` refers to. If it’s just 1, the implementation is simple, but after deletion, there's an issue with pagination. I can investigate it further, but wouldn't it be better to stay on the current page where the action was performed, and only move to the previous page if the deleted group was the last one on that page?

Reply:
agree with you, let's leave it as you currently done it